install/kubernetes/helm/istio/values.yaml

```diff
@@ -330,7 +330,12 @@ grafana:
     type: ClusterIP
     externalPort: 3000
     internalPort: 3000
+  security:
+    enabled: true
+    adminUser: admin
```

Comment:
I'm happy to remove the ingress.

I'm less clear on not having `enabled: true`. Since `security.adminUser` will always be set, won't this force user/pass on everyone?

Reply:
ingress removed.